ceno_zkvm/src/bin/e2e.rs

```diff
@@ -94,6 +101,17 @@ fn main() {
     let elf_bytes = fs::read(&args.elf).expect("read elf file");
     let mut vm = VMState::new_from_elf(platform.clone(), &elf_bytes).unwrap();
 
+    tracing::info!("Loading private input file: {:?}", args.private_input);
```

Comment:
On the discussion about private IO, someone suggested to use the term 'hint' throughout?

('Private input' works perfectly fine for me; but I remember some people having strong opinions on this.)

Reply:
Yeah, I favor `hint` over `private input`. Some places that use this terminology:
1. https://docs.cairo-lang.org/hello_cairo/program_input.html#hints
2. https://github.com/succinctlabs/sp1/blob/dev/crates/zkvm/entrypoint/src/syscalls/io.rs#L61

Reply:
Thanks for the ref. If the term "hint" is understood I’ll rename things to that.

Reply:
No worries, you can do that after you finish the sproll-evm test.

Reply:
https://github.com/scroll-tech/ceno/pull/636